### ISSUE TITLE
Track issues related to Classic sandboxes

### ISSUE THREAD

**Issue by apaskulin** [sensu/sandbox#10]
This issue tracks corrections that need to be made to the Classic sandboxes.

- `NOTE: The client gets its name from the sensu.name attribute configured as part of sandbox setup. You can change the client name using sudo nano /etc/sensu/uchiwa.json.` is probably incorrect.
- This could just be my setup, but I get a lot of these errors when starting the sandboxes:
```
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Connection reset. Retrying...
```

**Comment by jspaleta** [comment]
so for the error messages is that during the provisioning step while its installing packages?

its possible there are some dead mirrors in the CentOS mirror list,  Most of the somewhat disturbing warnings i see in the vagrant provisioning are related to package installation. Vagrant highlights all the messages thrown to stderr  as part of the provisioning script.

**Comment by apaskulin** [comment]
Fixed by #17 and #18
